src/main/java/Task.java

```diff
@@ -0,0 +1,22 @@
+public class Task {
+    protected String content;
+    protected Boolean isDone;
+
+    public Task(String content) {
+        this.content = content;
+        isDone = false;
+    }
+
+    public String TaskStatus() {
+        return (isDone? "X":" ");
```

Comment:
remember to leave spacing between certain punctuations :)

(isDone? "X":" ") -> (isDone ? "X" : " ")